lib/actions/install.js

```diff
@@ -101,12 +101,14 @@ install.installDependencies = function (options) {
     };
   }
 
-  options = _.defaults(options || {}, {
-    bower: true,
-    npm: true,
-    skipMessage: false,
-    callback: function () {}
-  });
+  if (!options) {
```

Comment:
Ok I understand this line yet. In the last commit I delete the npm: true by default.  The user will have to set npm: or yarn: true depends to the manager that want to use. Other posible solutions is leave option as it is and when the user want use yarn, put yarn: true and npm: false. who is the better option?